src/registrar/tests/common.py

```diff
@@ -93,6 +93,73 @@ def less_console_noise(output_stream=None):
             output_stream.close()
 
 
+class GenericTestHelper(TestCase):
+    """A helper class that contains various helper functions for TestCases"""
+
+    def __init__(self, admin, model=None, url=None, user=None, factory=None, **kwargs):
+        """
+        Parameters:
+            admin (ModelAdmin): The Django ModelAdmin instance associated with the model.
+            model (django.db.models.Model, optional): The Django model associated with the admin page.
+            url (str, optional): The URL of the Django Admin page to test.
+            user (User, optional): The Django User who is making the request.
+            factory (RequestFactory, optional): An instance of Django's RequestFactory.
+        """
+        super().__init__()
+        self.factory = factory
+        self.user = user
+        self.admin = admin
+        self.model = model
+        self.url = url
+
+    def assert_table_sorted(self, o_index, sort_fields):
```

Comment:
This doesn't have to be here or exist at all, it just dramatically simplifies the test cases and I think it provides a good backbone for future expansion, if we want it. Its intentionally pretty bare bones.

I know that generally we try to avoid architectural complexity for our test cases, so if this violates that principle I can take this out and migrate these changes to each test manually.